app/client-v2/src/__tests__/views/HomeView.spec.ts

```diff
@@ -198,4 +199,116 @@ describe("HomeView ", () => {
       expect(screen.getByText(/Error renaming project/i)).toBeVisible();
     });
   });
+
+  describe("Deleting projects", () => {
+    describe("When the user confirms the deletion", () => {
+      it("should send a request to delete project", async () => {
+        const fetchSpy = vi.spyOn(window, "fetch");
+        renderComponent();
+
+        await screen.findByRole("cell", {
+          name: MOCK_PROJECTS[0].name
+        });
+
+        const firstDeleteButton = await screen.getByRole("button", {
+          name: new RegExp(`delete ${MOCK_PROJECTS[0].name}`, "i")
+        });
+        await userEvent.click(firstDeleteButton);
+
+        // Mock a different response to 'get' (and, later, assert that this different set of projects is listed) to verify that get is called a second time
```

Comment:
Yeah, good idea. I guess an alternative would be to use fetchSpy again. But this way you also check that it gets rendered correctly.